health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/Attendance.js

```
@@ -0,0 +1,339 @@
+              />
+            ]}
+            onOverlayClick={() => setAssignUsersPopup(false)}
+            footerChildren={[
+              <Button
+                type="button"
+                size="large"
+                variation="secondary"
+                label={t("REGISTER_CLOSE")}
+                onClick={() => setAssignUsersPopup(false)}
+                style={{ minWidth: "200px" }}
+              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)